internal/website/content/howto/docstore/_index.md

```diff
@@ -12,3 +14,350 @@ with Docstore in the Go CDK.
 
 
 <!--more-->
+
+## Opening a Collection {#opening}
+
+The first step in using Docstore is connecting to your
+document store provider. Every document store provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*docstore.Collection`][] type.
+
+While every provider has the concept of a primary key that uniquely
+distinguishes a document in a collection, each one specifies that key in its own
+way. To be portable, Docstore requires that the key be part of the document's
+contents. When you open a collection using one of the functions described here,
+you specify how to find the provider's primary key in the document.
+
+
+[`*docstore.Collection`]: https://godoc.org/gocloud.dev/docstore##Collection
+
+<!--more-->
+
+### Constructors versus URL openers
+
+The easiest way to open a collection is using [`docstore.OpenCollection`][] and
+a URL pointing to the topic, making sure you ["blank import"][] the driver
+package to link it in. See [Concepts: URLs][] for more details. If you need
+fine-grained control over the connection settings, you can call the constructor
+function in the driver package directly (like `mongodocstore.OpenCollection`).
+This guide will show how to use both forms for each document store provider.
```

Comment:
This sentence got replaced in the merge by:

See the [guide below][] for usage of both forms for each supported provider. 
+ link to #services section